snowplow_tracker/events/structured_event.py

```diff
@@ -108,7 +114,7 @@ def label(self) -> Optional[str]:
         return self.payload.nv_pairs.get("se_la")
 
     @label.setter
-    def label(self, value: Optional[str]):
+    def label(self, value: str):
```

Comment:
I think that we should still allow the `label` and `property_` and `value` to be optional, that should not break the types as `Payload.add()` handles the `None` case.

Reply:
I'm coming back around to agreeing that they should be Optional, I was trying to think of a use case of allowing `None`. Maybe if it has been set to a value, and one wishes for it to be un-set?. Possibly a bit of an unconventional use-case but yeah I don't see why we should disallow it. I'll make them optional again.